tests/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+file(GLOB YS_TESTS "${CMAKE_CURRENT_SOURCE_DIR}/*/*.ys")
+file(GLOB TCL_TESTS "${CMAKE_CURRENT_SOURCE_DIR}/*/*.tcl")
```

Comment:
Avoid use of globs in CMake. If you use a glob, when you add a new test and re-run CMake, it won't see it and everything will appear green even though the code suggests it should be red. For the same reason I haven't used globs for the sources.

Reply:
Is there a way to make it pick up all tests without listing them?

Reply:
No, listing them is the whole point.